### PR TITLE
Add Big Local News

### PR DESCRIPTION
**Pull request by mitchelljkotler**

### CODE REVIEW

File: squarelet/templates/core/component/services.html [hunk 1]
```diff
@@ -52,20 +52,20 @@
...
+          <img src="{% static 'assets/biglocalnews.svg' %}">
         </div>
-        <div class="_cls-serviceDescription">
-          {% blocktrans %}A task-oriented chat bot for journalists using Slack{% endblocktrans %}
+        <div class="_cls-serviceInfo">
+          <div class="_cls-serviceName">
+            {% trans 'Big Local News' %}
+          </div>
+          <div class="_cls-serviceDescription">
+            {% blocktrans %}Big Local News description here.{% endblocktrans %}
```
Comment (mitchelljkotler) [inline squarelet/templates/core/component/services.html:65]: @palewire Need to insert a short description here

Comment (palewire) [reply squarelet/templates/core/component/services.html:65]: The standard Big Local News tagline is:

> Data, tools and collaborations that empower journalists to better cover their community. A project at Stanford University.

If we're looking to save space, we could go with:

> Data, tools and collaborations that empower journalists to better cover their community.

That work for you?

Comment (mitchelljkotler) [reply squarelet/templates/core/component/services.html]: Perfect

**Comment by palewire** [comment]
Here's what this is looking like in my local testing. 
Screenshot from 2022-06-02 13-19-03
Screenshot from 2022-06-02 13-16-40

**Pull request closed without merging**